### PR TITLE
Fixing some SSO error messages. 

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: api/user.go [hunk 1]
```diff
@@ -1462,6 +1462,9 @@ func AuthorizeOAuthUser(service, code, state, redirectUri string) (io.ReadCloser
 		return nil, nil, model.NewAppError("AuthorizeOAuthUser", "Token request failed", err.Error())
 	} else {
 		ar = model.AccessResponseFromJson(resp.Body)
+		if ar == nil {
+			return nil, nil, model.NewAppError("AuthorizeOAuthUser", "Bad responce from token request", "")
```
Comment (jwilander) [inline api/user.go]: Type "responce" -> "response"

**Comment by jwilander** [comment]
Other than typo, +1

**Comment by crspeller** [comment]
Typo fixed.

**Comment by hmhealey** [comment]
Looks typo free. Ready to merge once travis does it's thing.

**Pull request merged**